tests/foreman/ui/test_partitiontable.py

```diff
@@ -134,134 +132,6 @@ def test_positive_create_with_audit_comment(self):
                     )
                     self.assertIsNotNone(self.partitiontable.search(name))
 
-    @run_only_on('sat')
-    @tier2
-    def test_positive_create_default_for_organization(self):
-        """Create new partition table with enabled 'default' option. Check
-        that newly created organization has that partition table assigned to it
-
-        :id: 91c64054-cd0c-4d4b-888b-17d42e298527
-
-        :expectedresults: New partition table is created and is present in the
-            list of selected partition tables for any new organization
-
-        :CaseLevel: Integration
-        """
-        name = gen_string('alpha')
-        org_name = gen_string('alpha')
-        with Session(self) as session:
-            make_partitiontable(
-                session,
-                name=name,
-                template_path=PARTITION_SCRIPT_DATA_FILE,
-                default=True,
-            )
-            self.assertIsNotNone(self.partitiontable.search(name))
-            entities.Organization(name=org_name).create()
-            self.org.search_and_click(org_name)
-            session.nav.click(tab_locators['context.tab_ptable'])
-            # Item is listed in 'Selected Items' list and not 'All Items' list.
-            self.assertIsNotNone(
-                session.nav.wait_until_element(
-                    common_locators['entity_deselect'] % name)
-            )
-
-    @run_only_on('sat')
-    @tier2
-    def test_positive_create_non_default_for_organization(self):
```

Comment:
that's probably fine that you rename tests when you port them to have better readability and etc, but it is not okay to change their ids as Polarion in general will not be able to handle such changes in a easy way and require bunch of additional actions to be performed. It was not a joke when I asked whether you ported more than 2 tests as I cannot find same ids in my search procedure